Gruntfile.js

```diff
@@ -128,6 +128,7 @@ module.exports = function(grunt) {
                 },
                 output: {
                     path: 'bin-debug/',
+                    publicPath: '/jwplayer/bin-debug/',
```

Comment:
We will revisit this soon, Kyle made a story in pivotal for this : https://www.pivotaltracker.com/story/show/91134114